swagger.yml

```diff
@@ -2113,21 +2137,49 @@ paths:
                             type: boolean
                           Number:
                             type: integer
+                            description: 'The Page number '
```

Comment:
Today I noticed that some part of the swagger don't use the  '#/components/schemas/PageMetadata' component...

```
Metadata:
                        $ref: '#/components/schemas/PageMetadata'
```

Reply:
ok is that a problem ? what do you want to do ?

Reply:
1) Move the description to the #/components/schemas/PageMetadata
2) replace the PageMetadata schema with $ref: '#/components/schemas/PageMetadata'

Reply:
Yes sorry I didn't see your email. Done !